dbt_platform_helper/providers/vpc.py

```diff
@@ -77,8 +77,8 @@ def _get_vpc_id_by_name(self, vpc_name: str) -> str:
         return vpc_id
 
     def _get_security_groups(self, app: str, env: str, vpc_id: str) -> list:
-
         vpc_filter = {"Name": "vpc-id", "Values": [vpc_id]}
+        # TODO this is copilot specific make generic or set this value for the env security group
```

Comment:
If this is for a later ticket, can you raise a ticket for it and reference the ticket number in this comment.